docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -146,7 +146,186 @@ The file to export the slashing protection database to.
+is specified using [`--data-base-path`](#data-base-path-data-path).
+
+### eth1-deposit-contract-address
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=<ADDRESS>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=0x77f7bED277449F51505a4C54550B074030d989bC
+    ```
+
+Ethereum 1.0 address of the deposit contract. Only required when creating a custom network.
+
+### initial-state
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --initial-state=<STRING>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --initial-state=/home/me/genesis.ssz
+    ```
+
+Path or URL to an SSZ-encoded state file. The state file can be used to specify the genesis state,
+or a [recent finalized checkpoint state from which to sync].
+
+This option does not need to be specified if the initial state is provided by the network specified
+using the [`--network`](#network) option.
+
+### network
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --network=<NETWORK>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --network=mainnet
+    ```
+
+Predefined network configuration. Accepts a predefined network name, or file path or URL
+to a YAML configuration file. The default is `mainnet`.
+
+Possible values are:
+
+| Network   | Chain   | Type        | Description                                      |
+|-----------|---------|-------------|--------------------------------------------------|
+| `mainnet` | Eth 2.0 | Production  | Main network.                                    |
+| `minimal` | Eth 2.0 | Test        | Used for local testing and development networks. |
+| `pyrmont` | Eth 2.0 | Test        | Multi-client testnet.                            |
+| `prater`  | Eth 2.0 | Test        | Multi-client testnet.                            |
+
+Predefined networks can provide defaults such the initial state of the network,
+bootnodes, and the address of the Ethereum 1.0 deposit contract.
+
+### slot
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --slot=<INTEGER>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --slot=1028
+    ```
+
+Updates slashing protection files to contain the specified slot as a minimum. The value should be a future
+slot, or after when the validators stopped performing duties.
+
+!!! note
+
+    This can be automatically calculated for most networks, and is generally not required.
+
+### update-all-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --update-all-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --update-all-enabled=true
+    ```
+
+Enables slashing protection records to be updated. Defaults to `true`.
```

Comment:
```
      --update-all-enabled[=<BOOLEAN>]
                            All validator slashing protection records should be
                              updated.
                              Default: false
```

default behaviour: only update records that were corrupted.
This flag enables rewriting all slashing protection files, and it defaults to false. Basically, if a file looks valid, it will still be rewritten, you might use this if you're moving validators and want to guarantee that they can't start working before a specific slot.